site/_data/docs/extensions/toc.yml

```diff
@@ -44,7 +44,6 @@
     - url: /docs/extensions/mv3/service_workers
     - url: /docs/extensions/mv3/user_interface
     - url: /docs/extensions/mv3/tut_debugging
-    - url: /docs/extensions/mv3/samples
 - title: i18n.docs.extensions.service_workers
```

Comment:
```suggestion
    - url: /docs/extensions/samples
- title: i18n.docs.extensions.service_workers
```

Reply:
Hmm... I'm confused... Do we really need two places where we link to the samples page? 🤔


![Screenshot 2023-09-26 at 12 39 33](https://github.com/GoogleChrome/developer.chrome.com/assets/37001393/314dbe21-7f09-43f9-a4cc-39abc5f58c3e)

Reply:
My change isn't doing that. That's an existing problem we need to fix. It was probably done Oliver's new page was added. My requested change updates the left-had navigation in the content proper.

Reply:
I'll take care of that problem.

Reply:
I just figured out my mistake.